x-pack/plugins/siem/public/app/home/home_navigations.tsx

```diff
@@ -58,4 +58,11 @@ export const navTabs: SiemNavTab = {
     disabled: false,
     urlKey: 'case',
   },
+  [SiemPageName.management]: {
```

Comment:
Ok - long weekend and I forgot about this code here. I have an Enum further below that I will  adjust to use some of the data here (like `id`, `urlKey`)